pyproject.toml

```diff
@@ -91,12 +91,18 @@ tensorboardX = ["tensorboardX"]
 vllm = ["vllm", "ray", "more_itertools"]
 quality = ["ruff==v0.2.2","pre-commit"]
 tests = ["pytest==7.4.0"]
-dev = ["lighteval[accelerate,quality,tests]"]
+dev = ["lighteval[accelerate,quality,tests,multilingual]"]
```

Comment:
Multilingual should not be a part of dev as it's not needed to develop on lighteval
```suggestion
dev = ["lighteval[accelerate,quality,tests]"]
```

Reply:
I need to install it during the tests tho

Reply:
Hm good question, I'll let @NathanHB decide on this one